examples/vision/StableDiffusion/quantize_StableDiffusion.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+from quanto import quantize, freeze, qint4, qint8, qfloat8_e4m3fn
+import torch
+import torch.utils.benchmark as benchmark
+from diffusers import DiffusionPipeline
+
+CKPT = "runwayml/stable-diffusion-v1-5"
+NUM_INFERENCE_STEPS = 50
+WARM_UP_ITERS = 5
+PROMPT = "ghibli style, a fantasy landscape with castles"
+
+TORCH_DTYPES = {"fp32": torch.float32, "fp16": torch.float16, "bf16": torch.bfloat16}
+UNET_DTYPES = {"fp8": qfloat8_e4m3fn, "int8": qint8, "int4": qint4}
```

Comment:
`qfloat8` is a short alias for `qfloat8_e4m3fn`.
```suggestion
UNET_DTYPES = {"fp8": qfloat8, "int8": qint8, "int4": qint4}
```
Also, I would add `"none": None` so that the unet is not quantized, allowing to comparisons with the unquantized version of the model.